database/migrations/2018_06_14_212502_change_default_name_order_user_table.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+
+class ChangeDefaultNameOrderUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function ($table) {
+            $table->string('name_order')->default('firstname_lastname_nickname')->change();
```

Comment:
This has nothing to do with the UTC timezone, right ?

Reply:
No indeed. It's a bug fix for something else. I should put it out and create a separate PR for this.

Reply:
OK I have moved it in #1585